staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5215,19 +5215,8 @@ type EventSeries struct {
 	Count int32 `json:"count,omitempty" protobuf:"varint,1,name=count"`
 	// Time of the last occurrence observed
 	LastObservedTime metav1.MicroTime `json:"lastObservedTime,omitempty" protobuf:"bytes,2,name=lastObservedTime"`
-	// State of this Series: Ongoing or Finished
-	// Deprecated. Planned removal for 1.18
-	State EventSeriesState `json:"state,omitempty" protobuf:"bytes,3,name=state"`
```

Comment:
@liggitt - the test failures are relevant to this change. Does it mean we can't remove those fields?
Or should I fix the tests?

Reply:
per https://github.com/kubernetes/kubernetes/pull/75987#issuecomment-479309114, go ahead and update the tests (run with `UPDATE_COMPATIBILITY_FIXTURE_DATA=true`) and capture the test output to include as a diff